features/font-variation-settings.yml

```
@@ -0,0 +1,9 @@
+name: Font variation settings
+description: 'The `font-variation-settings` CSS property sets an "axis of variability" on a variable font, such as weight, optical size, or a custom axis defined by the typeface designer. When possible, use other CSS font properties, such as `font-weight: bold`.'
```

Caniuse uses a different name for this feature ("Variable fonts"). This is a good place for an "also known as" at the end.

```suggestion
description: 'The `font-variation-settings` CSS property sets an "axis of variability" on a variable font, such as weight, optical size, or a custom axis defined by the typeface designer. When possible, use other CSS font properties, such as `font-weight: bold`. Also known as variable fonts.'
```